<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(240, 240, 240,50%);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>700</width>
     <height>961</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 227, 231);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;苹方 中等&quot;;
	font-size: 14px;</string>
   </property>
   <property name="text">
    <string>调试界面</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:2px solid #4473c5;;
border-radius:5px;
margin-top: 2ex; 
	font-family:&quot;苹方 粗体&quot;;
	font-size:20px;
	color:#4473c5;

}
QGroupBox::title {
      subcontrol-origin: margin;
       left:10px;
  }</string>
   </property>
   <property name="title">
    <string>寸部调试</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;苹方 中等&quot;;
	font-size: 22px;</string>
    </property>
    <property name="text">
     <string>寸部压力梯度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cunPre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
padding:5px;	
font-family: &quot;苹方 中等&quot;;
	font-size: 12px;
	color: #686868;
border-radius: 6px;
	
	border-image: url(:/image/loadafter/buchong/shexingkuang.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>压力值从大到小排列，以,隔开，最多支持9个梯度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startcun">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>开始加压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startcun5s">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>加压5秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopfastcun">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止快放气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopslowcun">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止慢放气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopcun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止气泵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fastcun5s">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>快放气5秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_slowcun5s">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>慢放气5秒</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>lineEdit_cunPre</zorder>
   <zorder>pushButton_stopfastcun</zorder>
   <zorder>pushButton_stopslowcun</zorder>
   <zorder>pushButton_stopcun</zorder>
   <zorder>pushButton_fastcun5s</zorder>
   <zorder>pushButton_slowcun5s</zorder>
   <zorder>pushButton_startcun</zorder>
   <zorder>pushButton_startcun5s</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:2px solid #4473c5;;
border-radius:5px;
margin-top: 2ex; 
	font-family:&quot;苹方 粗体&quot;;
	font-size:20px;
	color:#4473c5;

}
QGroupBox::title {
      subcontrol-origin: margin;
       left:10px;
  }</string>
   </property>
   <property name="title">
    <string>关部调试</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;苹方 中等&quot;;
	font-size: 22px;</string>
    </property>
    <property name="text">
     <string>关部压力梯度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_guanPre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
padding:5px;	
font-family: &quot;苹方 中等&quot;;
	font-size: 12px;
	color: #686868;
border-radius: 6px;
	
	border-image: url(:/image/loadafter/buchong/shexingkuang.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>压力值从大到小排列，以,隔开，最多支持9个梯度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startguan">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>开始加压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startguan5s">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>加压5秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopfastguan">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止快放气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopslowcun_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止慢放气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopguan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止气泵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fastguan5s">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>快放气5秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_slowguan5s">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>慢放气5秒</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>640</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:2px solid #4473c5;;
border-radius:5px;
margin-top: 2ex; 
	font-family:&quot;苹方 粗体&quot;;
	font-size:20px;
	color:#4473c5;

}
QGroupBox::title {
      subcontrol-origin: margin;
       left:10px;
  }</string>
   </property>
   <property name="title">
    <string>尺部调试</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;苹方 中等&quot;;
	font-size: 22px;</string>
    </property>
    <property name="text">
     <string>尺部压力梯度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chiPre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
padding:5px;	
font-family: &quot;苹方 中等&quot;;
	font-size: 12px;
	color: #686868;
border-radius: 6px;
	
	border-image: url(:/image/loadafter/buchong/shexingkuang.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>压力值从大到小排列，以,隔开，最多支持9个梯度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startchi">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>开始加压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startchi5s">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>加压5秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopfastchi">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止快放气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopslowchi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止慢放气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopchi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>停止气泵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fastchi5s">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>快放气5秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_slowchi5s">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 18px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
    </property>
    <property name="text">
     <string>慢放气5秒</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>980</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/image/loadafter/1/xiayibuweixuanzhong.png);
border-radius: 22px;
color: rgb(24, 24, 24);
font-family: &quot;苹方 粗体&quot;;
	font-size: 24px;}
QPushButton:focus,QPushButton:hover{
border-image: url(:/image/loadafter/tab/2/组25.png);
}
	
QPushButton:pressed{
url(:/image/loadafter/1/xiayibuanzhong.png)
}</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_command">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>900</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
padding:5px;	
font-family: &quot;苹方 中等&quot;;
	font-size: 12px;
	color: #686868;
border-radius: 6px;
	
	border-image: url(:/image/loadafter/buchong/shexingkuang.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>900</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;苹方 中等&quot;;
	font-size: 22px;</string>
   </property>
   <property name="text">
    <string>下发命令：</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>lineEdit_command</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
